### PR TITLE
refactor(query-engine): yeet substitute_params

### PR DESCRIPTION
**Pull request by yakkomajuri**
Seeing how tests go first

### CODE REVIEW

File: posthog/queries/person_distinct_id_query.py [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 
 def get_team_distinct_ids_query(team_id: int) -> str:
-    from posthog.client import substitute_params
 
-    return substitute_params(GET_TEAM_PERSON_DISTINCT_IDS, {"team_id": team_id})
+    # make sure team_id is actually an int so we can safely interpolate into the query
```
Comment (yakkomajuri) [inline posthog/queries/person_distinct_id_query.py:6]:
```suggestion
    # ensure team_id is actually an int so we can safely interpolate into the query
```

**Comment by yakkomajuri** [comment]
all good otherwise? @macobo

**Review by macobo — APPROVED** [review]
All good assuming tests pass

**Review by mariusandra — APPROVED** [review]
👍 

One flakey build test running on 18+ min, will merge regardless.

**Pull request merged**